pkg/utils/addresses.go

```diff
@@ -94,6 +101,16 @@ func ValidNodeAddress(address netlink.Addr) bool {
 		return false
 	}
 
+	// Ignore the IPv6 OVNK masquerade subnet. We should never see an error here as this is a constant.
+	// If we get an error, panic.
+	_, netip, err := net.ParseCIDR(V6MasqueradeSubnet)
+	if err != nil {
+		panic(err)
+	}
+	if netip.Contains(address.IP) {
```

Comment:
I still don't think we can do this unconditionally. The masquerade subnet is a valid address range in non-OVNK CNIs.

Reply:
I removed it altogether for this PR